### DISCUSSION TITLE
Is it possible to use Node JS v20 with Mastodon?

### DISCUSSION THREAD

**Discussion by corico44** [mastodon/mastodon#27019] [General]
Reviewing the Mastodon documentation: https://github.com/mastodon/mastodon/releases, it indicates that the node version must be lower than version 18 but I wanted to confirm if it is possible to use Node JS v18 or v20 with Mastodon.

**Comment by BossMAN559** [comment]
i have been using version 20, i just need to run 
export NODE_OPTIONS=--openssl-legacy-provider
before i recompile the assets

**Reply by Fluqzy** [reply]
hey, can you tell me which system distro your on and what exact version you have running?

**Reply by BossMAN559** [reply]
Distributor ID: Ubuntu
Description:    Ubuntu 23.04
Release:        23.04
Codename:       lunar

node version V20.5.1

**Comment by renchap** [comment]
https://github.com/mastodon/mastodon/releases/tag/v4.2.0 correctly indicates that Mastodon 4.2.0 supports Node 16+ (tested with Node 16, 18 and 20).

Mastodon 4.1 supported Node 14 & 16.

**Reply by zhoukeca** [reply]
how about 22?
